CHANGELOG-LATEST.md

```diff
@@ -1,3 +1,26 @@
 # @financialforcedev/orizuru
 
 ## Latest changes (not yet released)
+
+## 9.0.0-0 PREMAJOR
+
+- The `addRoute` method can now add synchronous APIs.
+	- Set the `synchronous` property to `true` in the route options to do this.
+	- All incoming messages are still validated against the Apache Avro schema but are not published to a message queue.
+	- Synchronous request logic should be added to the `responseWriter` property.
```

Comment:
Maybe should add that the validated request object is in the body property of the request?

Reply:
Added